deny.toml

```diff
@@ -60,12 +60,15 @@ skip-tree = [
     # wait for criterion to upgrade
     { name = "itertools", version = "=0.10.5" },
 
-    # wait for backtrace and multiple dependents to upgrade
-    { name = "miniz_oxide", version = "=0.6.2" },
-
     # wait for h2 and tower to upgrade
     { name = "indexmap", version = "=1.9.3" },
 
+    # wait for rocksdb to upgrade
+    { name = "bindgen", version = "=0.65.1" },
+
+    # wait for tempfile to upgrade
+    { name = "rustix", version = "=0.37.23" },
+
     # ZF crates
 
     # wait for indexmap, toml_edit, serde_json, tower to upgrade
```

Comment:
`windows-sys` can also be removed:
https://github.com/ZcashFoundation/zebra/actions/runs/5593292448/jobs/10226778670?pr=7255#step:5:17

```
warning[unmatched-skip-root]: skip tree root was not found in the dependency graph
    ┌─ /github/workspace/deny.toml:116:5
    │
116 │     { name = "windows-sys", version = "=0.42.0" },
    │     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no crate matched these criteria
```

Reply:
Removed in https://github.com/ZcashFoundation/zebra/pull/7255/commits/19b8f89fd240e18d44d7b529b13edfa90d1c51ea.

Reply:
The duplicate dependency job tests all features, so it doesn't seem to be needed:
https://github.com/ZcashFoundation/zebra/actions/runs/5593536987/jobs/10227293175?pr=7255#step:5:14